storage/migrations/05_evm_runtime_bytecode.up.sql

```diff
@@ -0,0 +1,39 @@
+BEGIN;
+
+-- A schema for keeping track of analyzers' internal state/progess.
+-- TODO: Move chain.processed_blocks and chain.*_analysis tables here.
+CREATE SCHEMA IF NOT EXISTS analysis;
```

Comment:
@pro-wh We discussed DB wiping. This new schema will be covered automatically; we wipe [all schemas except the built-in ones](https://github.com/oasisprotocol/oasis-indexer/blob/435fe753b227b8c8036b881cd8b838321baf6ab6/storage/postgres/client.go#L218-L218)

Reply:
o good